tests/unittests/quantizationTest.cpp

```diff
@@ -1111,6 +1111,154 @@ TEST(Quantization, quantizeFunctionConvertConstant) {
   EE.run(ctx);
 }
 
+/// Check that the slice node doesn't change the quantization parameters between
+/// its input and output.
+TEST(Quantization, quantizeSlice) {
+  ExecutionEngine EE;
+  Context ctx;
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  auto *input = mod.createPlaceholder(ElemKind::FloatTy, {4}, "input", true);
+  ctx.allocate(input)->init(Tensor::InitKind::Xavier, 3, mod.getPRNG());
+
+  auto *slice = F->createSlice("slice", input, {2}, {3});
+  auto *save = F->createSave("ret", slice);
+  auto *result = save->getPlaceholder();
+  ctx.allocate(result);
+
+  // Note that we are creating quantization info even for nodes that will not be
```

Comment:
I think this comment is not correct in this case?

Reply:
😱 have been caught doing copy/paste!

Reply:
Done!